examples/errors/JumpTableLabelNotFound.huff

```diff
@@ -0,0 +1,19 @@
+#define macro MAIN() = takes(0) returns(0) {
+    ANOTHER()
+    asterix:
```

Comment:
if you used dests instead of offsets, this can be allowed

Reply:
Has nothing to do with that, could allow it regardless, just a choice I made to have tables be more explicit.

Reply:
The table is already explicit enough I think; it doesn't need to be limited